resources/linux-config-aarch64

```diff
@@ -1402,7 +1402,9 @@ CONFIG_NVME_MULTIPATH=y
 # CONFIG_MISC_RTSX_PCI is not set
 # CONFIG_HABANA_AI is not set
 # CONFIG_UACCE is not set
-# CONFIG_PVPANIC is not set
+CONFIG_PVPANIC=y
+CONFIG_PVPANIC_MMIO=y
```

Comment:
Do you need this one? Does PCI depend on it?

Reply:
No :)
We need pvpanic-pci only now, I will fix that immediately.